scripts/scripts.js

```diff
@@ -212,6 +232,18 @@ async function loadPage() {
   const {
     loadArea, loadLana, setConfig, createTag, getMetadata, getLocale,
   } = await import(`${LIBS}/utils/utils.js`);
+
+  let eventUtils;
+  const eventMD = getMetadata('event-id');
+
+  if (eventMD) {
+    try {
+      eventUtils = await import(`${EVENT_LIBS}/libs.js`);
+    } catch (e) {
+      window?.lana?.log(`Could not import event-libs. ${e}`, { tags: 'event-libs' });
```

Comment:
Lana isn't loaded until later down in the file so would this ever get triggered?